enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/atomicbroadcast/multipaxos/context/HeartbeatContextImpl.java

```diff
@@ -141,6 +141,14 @@ public void notify( HeartbeatListener listener )
     @Override
     public void suspicions( InstanceId from, Set<InstanceId> suspicions )
     {
+        // Ignoring this message if it is the first received from a failed instance.
+        if ( isFailed( from ) )
```

Comment:
Changed the comment. It is not interesting if it is the the first or second or whatever. The point is that a failed instance is likely to have an out-of-date view of the other cluster members and incorrectly suspecting instances which are alive.